package.json

```
@@ -84,7 +83,7 @@
     "eslint-plugin-prettier": "^3.1.4",
     "fetch-mock": "^9.11.0",
     "graphql": "^15.4.0",
-    "graphql-relay": "^0.6.0",
+    "graphql-relay": "^0.670",
```

This was supposed to be `0.7.0` correct?
```suggestion
    "graphql-relay": "^0.7.0",
```

---

oops